data/talkactions/scripts/create_item.lua

```diff
@@ -48,5 +54,5 @@ function onSay(player, words, param)
 		end
 		player:getPosition():sendMagicEffect(CONST_ME_MAGIC_GREEN)
 	end
-	return false
+	return true
```

Comment:
Talkactions return `false` when the engine is not supposed to look further (e.g. try to find spells). They return `true` when the talkaction is unavailable, rather than failed, so that the game next tries to find a spell with the same words. That's why you can't do this change right now.